Client/Frontend/Browser/BrowserToolbar.swift

```diff
@@ -82,38 +82,38 @@ public class BrowserToolbarHelper: NSObject {
         toolbar.backButton.setImage(UIImage(named: "backPressed"), forState: .Highlighted)
         toolbar.backButton.accessibilityLabel = NSLocalizedString("Back", comment: "Accessibility Label for the browser toolbar Back button")
         //toolbar.backButton.accessibilityHint = NSLocalizedString("Double tap and hold to open history", comment: "")
-        let longPressGestureBackButton = UILongPressGestureRecognizer(target: self, action: "SELdidLongPressBack:")
+        let longPressGestureBackButton = UILongPressGestureRecognizer(target: self, action: #selector(BrowserToolbarHelper.SELdidLongPressBack(_:)))
```

Comment:
Nit: We might as well start renaming all of the SEL* selectors to just * since it's not needed anymore. At least now we have compile time checking to make sure we are calling a valid selector if anything goes awry.